change/@internal-react-composites-93739eae-6467-4523-af0e-4afbe6d8c5c9.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Add troubleshooting error bar to call composite config ",
```

Comment:
How do we get the network quality metric on the calling screen? As we aren't in a call yet do we get any UFD's this early from the calling SDK?